go/scanner/scanner.go

```diff
@@ -96,16 +106,18 @@ func (m MatchSubjectRegex) PrecertificateMatches(p *ct.Precertificate) bool {
 	return false
 }
 
-// Matches on issuer cn by regex
+// MatchIssuerRegex matches on issuer common name by regex
```

Comment:
In some places you use the abbreviation "CN", in others you write "common name". Best to consistently use one.